app/uk/gov/hmrc/trackingconsentfrontend/assets/src/ui/renderBanner.ts

```diff
@@ -50,10 +51,10 @@ const insertBanner = (userPreference: UserPreferences) => {
     insertAfterSkipLink(banner)
 }
 
-const renderBanner = (userPreference: UserPreferences) => {
+export const renderBanner = (_, userPreference: UserPreferences) => {
```

Comment:
Is underscore an unused `document`?  renderBanner uses `document` a few times, surely it should use the one that's provided to it?

Reply:
I actually think it's unnecessary passing a document in. It is fundamentally a global. I added the param purely to be consistent with the formHandler.